integration-test/src/test/resources/trino-ci-testset/testsets/lakehouse-iceberg/00000_create_table.sql

```diff
@@ -3,8 +3,13 @@ CREATE SCHEMA "test.gt_iceberg".gt_db2;
 CREATE TABLE "test.gt_iceberg".gt_db2.tb01 (
     name varchar,
     salary int
+) with (
+    partitioning = ARRAY['name'],
+    sorted_by = ARRAY['salary']
 );
```

Comment:
I suggest keeping the original test cases, unless you are reorganizing them.
Additionally, add some erroneous test cases to verify the error messages.